src/topology/locally_constant/algebra.lean

```diff
@@ -110,4 +117,29 @@ instance [ring Y] : ring (locally_constant X Y) :=
 instance [comm_ring Y] : comm_ring (locally_constant X Y) :=
 { .. locally_constant.comm_semiring, .. locally_constant.ring }
 
+variables (R : Type*)
```

Comment:
At least for `coe_smul` and `smul_apply`, this should be implicit.

I think it's probably fine to have it explicit for the isntances too:
```suggestion
variables {R : Type*}
```